smartling-files-api/src/main/java/com/smartling/api/files/v2/pto/FileType.java

```diff
@@ -37,7 +37,8 @@
     TMX("application/xml", false),                  // TMX translation memory file format
     XLIFF_CAT("application/xml", false),            // XLIFF for offline CAT integration
     DITA("application/xml", false),                 // DITA file format
-    PDF("application/octet-stream", false);         // PDF file format
+    PDF("application/octet-stream", false),         // PDF file format
+    FLUENT("text/plain", true);
```

Comment:
@RRagnar please confirm that on the FAPI side we will have the same FileType code

Reply:
Google shows lots of Fluent file types, let's comment it as "Mozilla Fluent localization system"